lib/shared/bucketing-assembly-script/index.cjs

```diff
@@ -0,0 +1,16 @@
+const { instantiate } = require('./build/bucketing-lib.debug.commonjs')
+
+const url = new URL("build/bucketing-lib.release.commonjs.wasm", import.meta.url)
+
+exports.instantiate = async () => {
+    return await instantiate(
+        await (async () => {
+            try {
+                const source = await globalThis.fetch(url)
+                return await globalThis.WebAssembly.compileStreaming(source);
+            }
+            catch { return globalThis.WebAssembly.compile(await (await import("node:fs/promises")).readFile(url)); }
+        })(), {
```

Comment:
why this empty object?

Reply:
just copied this from the ESM output the compiler was giving us 🤷